release_notes/48-release-notes.adoc

```diff
@@ -419,4 +420,24 @@ This release also addresses the following security vulnerabilities:
 * Flaw in libxml2 library (link:https://access.redhat.com/security/cve/CVE-2025-32415[CVE-2025-32415])
 * Perl standard library threads component flaw (link:https://access.redhat.com/security/cve/CVE-2025-40909[CVE-2025-40909])
 
+
+[id="about-this-release-483_{context}"]
+== About release 4.8.3
+
+*Release date* {ga-date-483}
+
+This release of {product-title-short} 4.8 provides the following bug fixes:
+
+//ROX-30498
+* Before this update, you might have experienced user service interruptions due to a consistently growing Central database that caused insufficient storage space, which triggered "no space left on device" errors and regular crash loop backoff's. With this release, you get proactive disk space management, which controls Central database storage growth and minimizes the occurrence of "no space left on device" errors.
```

Comment:
This does not exactly capture what the fix does at all.
//ROX-30498, ROX-30447
* Before this update, you might have experienced consistently growing Central database that caused constant resizing of the PVC or service interruptions due to encountering "no space left on device" in regards to the PVC storage.  The options for remedy prior to this release were minimal.  With this release, if the "hashes" table is the table of growth, the feature can be turned off by setting "ROX_HASH_FLUSH_INTERVAL=0".